comet-accumulo/src/main/java/org/renci/comet/accumuloops/AccumuloOperationsApiImpl.java

```diff
@@ -234,11 +234,18 @@ private static BatchWriter createBatchWriter(Connector conn, String table) {
 
         }
         catch (AccumuloSecurityException securityException) {
```

Comment:
I think we can remove AccumuloSecurtitException catch block since we are handling it in Exception catch block

Reply:
ok will clean it